buildpacks/ruby/src/steps/default_env.rs

```diff
@@ -38,12 +38,20 @@ pub(crate) fn default_env(
         .to_string();
 
     let layer_ref = context.uncached_layer(
-        layer_name!("env_defaults"),
+        layer_name!("venv"),
         UncachedLayerDefinition {
             build: true,
             launch: true,
         },
     )?;
+    let update_env = LayerEnv::new()
+        .chainable_insert(Scope::All, ModificationBehavior::Delimiter, "PATH", ":")
+        .chainable_insert(
+            Scope::All,
+            ModificationBehavior::Prepend,
+            "PATH",
+            context.app_dir.join("bin"),
+        );
```

Comment:
I realized that this works for "launch" but not for "build" i.e. running `rake assets:precompile`. We could either introduce a new layer or add this bit of logic in the `gems` layer, which I'm leaning towards.

PR is still okay to review, but know that I'm re-visiting where this logic lives